scripts/generate_tool_list_from_ga_workflow_files.py

```diff
@@ -56,13 +56,20 @@ def print_yaml_tool_list(tool_dictionary, output_file):
         F.write(yaml.safe_dump(tool_dictionary, default_flow_style=False))
     return
 
-if __name__ == "__main__":
-    options = _parse_cli_options()
+def generate_tool_list_from_workflow(workflow_files, panel_label, output_file):
+    """
+
+    :rtype: object
```

Comment:
Super-minor comment, `:rtype: object` would mean that this function returns an object,
but there's no return statement here.